src/ui/zcl_abapgit_gui_chunk_lib.clas.abap

```diff
@@ -116,6 +116,18 @@ CLASS zcl_abapgit_gui_chunk_lib DEFINITION
       RETURNING VALUE(ro_menu) TYPE REF TO zcl_abapgit_html_toolbar.
 
   PROTECTED SECTION.
+    CLASS-METHODS render_repo_top_commit_hash
+      IMPORTING
+        iv_html        TYPE REF TO zcl_abapgit_html
+        iv_repo_online TYPE REF TO zcl_abapgit_repo_online
+        iv_repo_url    TYPE zif_abapgit_persistence=>ty_repo-url
+      RAISING
+        zcx_abapgit_exception.
+    CLASS-METHODS get_commit_display_url
```

Comment:
Suggest to move it to zcl_abapgit_utils or some other utils. The chunk lib is about rendering, and this method is a "backend" calculation.